storage/client/README.md

```diff
@@ -0,0 +1,365 @@
+    --container-name javablobsettiererror15514606921543
+    --name javablobsettiererror15514606921543
+    --permissions rwdac
+    --expiry 2019-06-05
+```
+Alternatively, get the Account SAS Token from the Azure Portal.
+
+## Key concepts
+### URL format
+Queues are addressable using the following URL format:
+https://<storage account>.queue.core.windows.net/<queue>
+The following URL addresses a queue in the diagram:
+https://myaccount.queue.core.windows.net/images-to-download
+
+#### Resource URI Syntax
+For the storage account, the base URI for queue operations includes the name of the account only:
+```$xslt
+https://myaccount.queue.core.windows.net
+```
+For a queue, the base URI includes the name of the account and the name of the queue:
+```$xslt
+https://myaccount.queue.core.windows.net/myqueue
+```
+
+Each resource may also have components, which are expressed as part of the query string on the base URI. The syntax for the query string is `?comp=<component_name>`. 
+Currently, the following resource components are available for queue resources, as parameters on the URI.
+
+| Component   |      Supported resources      |  Description |
+|:----------:|:-------------:|------|
+| `list` |  Account | Enumerates the entities beneath this resource. On the account, returns the list of queues. |
+| `metadata` |    Queue   | Sets or returns queue properties, including user-defined metadata. |
+
+### Supported HTTP Operations
+Each resource supports operations based on the HTTP verbs GET, PUT, HEAD, and DELETE. 
+The verb, syntax, and supported HTTP version(s) for each operation appears on the reference page for each operation. 
+
+### Queue Names
+Every queue within an account must have a unique name. The queue name must be a valid DNS name, and cannot be changed once created. Queue names must confirm to the following rules:
+1. A queue name must start with a letter or number, and can only contain letters, numbers, and the dash (-) character.
+1. The first and last letters in the queue name must be alphanumeric. The dash (-) character cannot be the first or last character. Consecutive dash characters are not permitted in the queue name.
+1. All letters in a queue name must be lowercase.
+1. A queue name must be from 3 through 63 characters long.
+
+### Metadata Names
+Metadata for a queue resource is stored as name-value pairs. 
+Note that metadata names preserve the case with which they were created, but are case-insensitive when set or read. 
+If two or more metadata headers with the same name are submitted for a resource, the Queue service returns status code 400 (Bad Request).
+
+### Timeout
+A call to a Queue service API can include a server timeout interval, specified in the timeout parameter of the request URI. If the server timeout interval elapses before the service has finished processing the request, the service returns an error.
```

Comment:
The SDK doesn't have functionality for server side timeouts at this time, we should remove this section.

Reply:
Removed.